inference/synapse/local_infer.go

```diff
@@ -42,8 +67,23 @@ func (s *Synapse) getGasByInfoHash(modelInfoHash string) (gas uint64, err error)
 	)
 	modelJson, modelJson_err = s.config.Storagefs.GetFile(s.ctx, modelHash, SYMBOL_PATH)
 	if modelJson_err != nil || modelJson == nil {
-		log.Warn("GetGasByInfoHash: get file failed", "error", modelJson_err, "hash", modelInfoHash)
-		return 0, KERNEL_RUNTIME_ERROR
+		if cvmNetworkId == 43 &&
+			modelHash == "31f75c90e8fe1c5b16cbdc466dc6127487a92add" {
+			// TODO(ryt): find ways to get the NetwordId of `dolores`
+			// and append it to the judgment here
+			modelJson, modelJson_err =
+				hackFile(modelInfoHash, "data/symbol")
```

Comment:
If the correct file is no blockchain with hash ```123345...```, what you should do is only modify the``` hash``` before ```modelJson, modelJson_err = s.config.Storagefs.GetFile(s.ctx, modelHash, SYMBOL_PATH)``` , not reading file from a special directory.

Reply:
Alright, but I'm trying to figure out how to upload the hackfiles to block chain with a self-set hash value.
For example, 
wrong hash: `de58609743e5cd0cb18798d91a196f418ac25016`  mapped to correct hash  `asdcndefghijklmnopq..`

Reply:
Can you find the same model or data file (which can make the txs pass) in storage path with different hash from ```de58609743e5cd0cb18798d91a196f418ac25016 and 31f75c90e8fe1c5b16cbdc466dc6127487a92add```? I think same file may be uploaded several times in dolores.

Reply:
For map you can see ```synapse/results.go```, it is what we do before

Reply:
By file comparing using python parser, it seems that there is no other datafile `~/.cortex/dolores/storage/[hash]/data` matching with  `~/.cortex/dolores/storage/de58609743e5cd0cb18798d91a196f418ac25016/data`.
I'm comparing mainnet hash data files `~/.cortex/storage/[hash]/data`, and do some double checking.

Reply:
not same hash like ```de58609743e5cd0cb18798d91a196f418ac25016``` in file path name, but same md5(data)

Reply:
de58609743e5cd0cb18798d91a196f418ac25016:data (wrong hash)

Reply:
Yeah, that's what exactly had been done: data binary comparison using python command `filecmp.cmp(path1, path2, shallow=False)`.
Luckily, wrong hash `31f75c90e8fe1c5b16cbdc466dc6127487a92add` was found to have the same `params binary` and  `symbol json` with either `0d23bf93f21b80393cce77ad8021386458adb8fc` and `c816d85a023abacb687080ca57f580d4999b8b1d`.

Reply:
Great

Reply:
```c816d85a023abacb687080ca57f580d4999b8b1d``` can be downloaded from torrent, so it can be used as the hack hash of ```31f75c90e8fe1c5b16cbdc466dc6127487a92add```

Reply:
```0d23bf93f21b80393cce77ad8021386458adb8fc``` has been downloaded too

Reply:
Ok, I'm debugging on them now.

Reply:
![image](https://user-images.githubusercontent.com/22344498/85986780-53266880-ba1f-11ea-8c66-973d6d29873e.png)